karoo_gp/tree.py

```diff
@@ -128,7 +128,7 @@ def set_child(self, i_child, node, **kwargs):
         self.renumber()
 
     # In mutation, node types within each tuple can be swapped
-    swappable = [('terminal', 'constant'), ('operator', 'cond'), ('bool')]
+    swappable = [['terminal', 'constant'], ['operator', 'cond'], ['bool']]
```

Comment:
If these are pairs, using a list of tuples might be ok, however the last one only has one element...
The comment should also be updated.